net-proxy/byedpi/byedpi-0.12.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Bypass DPI SOCKS proxy"
+HOMEPAGE="https://github.com/hufrea/byedpi/"
+SRC_URI="https://github.com/hufrea/byedpi/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+src_compile() {
+	tc-export CC
+	sed -i 's/ -O.\b/ /' Makefile
+	# workaround for compiling without -O2
```

Comment:
Please mention that this is pulled from upstream at https://github.com/hufrea/byedpi/commit/3fee8d5aed122f34ec13637f5f4b1502d13cc923

Reply:
Actually no, I did it myself and then realize, that it's already in upstream

But ok

Reply:
Ah okay, it's fine then.